docs/attributes-registry/vcs.md

```diff
@@ -96,6 +98,18 @@ the `.git` extension.
 
 ---
 
+`vcs.provider.name` has the following list of well-known values. If one of them applies, then the respective value MUST be used; otherwise, a custom value MAY be used.
```

Comment:
Since we may use a custom value is there a reason to include `other_provider`?

Reply:
good point, I'll create a PR to remove it

Reply:
https://github.com/open-telemetry/semantic-conventions/pull/2020